source/atomics.tex

```diff
@@ -1350,8 +1350,15 @@
 
 \indexlibrary{\idxcode{atomic}}%
 \pnum
-The template argument for
-\tcode{T} shall be trivially copyable\iref{basic.types}. \begin{note} Type arguments that are
+The template argument for \tcode{T} shall
+meet the \tcode{CopyConstructible} and \tcode{CopyAssignable} requirements.
```

Comment:
```suggestion
meet the \oldconcept{CopyConstructible} and \oldconcept{CopyAssignable} requirements.
```

Reply:
@CaseyCarter The resolution for LWG3012 doesn't use the Cpp17 prefix here but I think you're right, so I've made the change.

Reply:
Yes, I should have clarified. The P/R wording for LWG3012 predates incorporation of P0898 into the WD that added all of the Cpp17 prefixes. LWG should have made the change in San Diego when they approved the P/R, but folks aren't used to the prefixes yet and I was too busy drafting to show up for issue processing.

Thanks!